Runtime/UI/Utils/SpriteStretch.cs

```diff
@@ -8,10 +8,8 @@ namespace Padoru.Core
     public class SpriteStretch : MonoBehaviour
     {
         [SerializeField] private SpriteRenderer spriteRenderer;
-        
         [SerializeField] private bool keepAspectRatio;
-
-        [SerializeField] private bool stretchOnStart;
+        [SerializeField] private bool stretchOnStartUsingMainCamera;
```

Comment:
This should be called `stretchOnStart` only, the current name suggests there are alternatives to the main camera when they are not. Although we could add a tool tip specifying it will use the main camera